map_gui/src/render/map.rs

```diff
@@ -39,8 +39,9 @@ pub struct DrawMap {
     pub show_zorder: isize,
 
     quadtree: QuadTree<ID>,
-    // Remember these so we can delete lanes.
+    // Remember these so we can modify the bounding box of some objects.
     lane_ids: HashMap<LaneID, aabb_quadtree::ItemId>,
+    intersection_ids: HashMap<IntersectionID, aabb_quadtree::ItemId>,
```

Comment:
Alternatively, we could combine `lane_ids`/`intersection_ids` into a single: `HashMap<ID , aabb_quadtree::ItemId>`

... orrr I can ask again if we add another type that needs to be reverse lookup'd

Reply:
Done! I get the feeling that we'll allow editing parking lots at some point. ;)